README.md

```diff
@@ -1 +1,3 @@
-# Analytics
\ No newline at end of file
+# Analytics
+
+link for the looker studio to answer the BQ 2.3: https://lookerstudio.google.com/s/j8pzo3OZxkg
```

Comment:
https://lookerstudio.google.com/s/j8pzo3OZxkg is not a public link! Check this:


Uploading Screen Recording 2024-03-22 at 16.03.09.mov…